Longest_Common_Subsequence/main.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Author: Yash Ladha
```

Comment:
Remove `Author` as this is an open sourced project and the file may go through changes/improvements (from others) later on. Also the git history will preserve the authorship so no credits are lost.